pages/polling/create.tsx

```diff
@@ -58,6 +58,31 @@ const CreateText = ({ children }) => {
     </Box>
   );
 };
+
+const NetworkSwitchModal = () => {
```

Comment:
This modal blocks the app if the user is not connected to Arbitrum. It doesn't allow to switch networks, but if you move to a different page and switch there, you get an error. Maybe we can add a button to trigger Wagmi's `switchChain(arbitrum)` from this modal